app/models/ability.rb

```diff
@@ -8,7 +8,10 @@ def initialize(user)
     if user.administrator?
       can :manage, :all
       can :create, Course
-      can :refresh, Course
+      cannot :refresh, Course
```

Comment:
Hmm. At this point it can be problematic for non-custom course. In those cases, straight refresh for the course can cause other courses from same template be non-synced. If needed, can create link for template refresh-instead actually..

This will be changed to more clear solution later, after all courses are migrated to have template.